doc/rst/technotes/gpu.rst

```diff
@@ -5,8 +5,10 @@
 GPU Programming
 ===============
 
-Chapel can be used to program GPUs. Currently  NVIDIA and AMD GPUs are
-supported. Support for Intel GPUs is planned but not implemented, yet.
+Chapel can be used to program GPUs. The `GPU Programming in Chapel series
```

Comment:
I'd probably expand this intro somewhat and put the emphasis first on what the technote is, then mention the blog series at the end. (I do think linking to the blog series early is good though as that's a resource some people may prefer).

---
Something like this:

Chapel enables developers to use parallelism at different levels: from intra-node multicore parallelism, to cross-node distributed parallelism, to GPUs. This technote serves as a reference on how to use Chapel to program GPUs. Specifically, it gives a quick overview of GPU programming, includes some a handful of examples, discusses system requirements and current limitations for GPU support, and delves into more details on some specific GPU related features.

Readers preferring a more tutorial like introduction to GPU support, may also wish to look at our blog series: <https://chapel-lang.org/blog/series/gpu-programming-in-chapel/>.